test/ueber_google_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule UeberauthTwitterTest do
```

Comment:
Copy + Pasta fail. 

![facepalm](http://s.lurkmore.to/images/thumb/c/c2/Not-again-picard2.jpg/64px-Not-again-picard2.jpg)